lsp-mode.el

```diff
@@ -6190,14 +6190,21 @@ information, for example if it doesn't support DocumentSymbols."
   :group 'lsp-imenu
   :type 'boolean)
 
+(defface lsp-signature-face '((t :height 0.8 :inherit shadow))
```

Comment:
If you name it lsp-signature-face we won't be able to use it in lsp-treemacs. Why not using the more generic name lsp-details-face as suggested?

Reply:
Is that really a good idea though? Details might be many different things. A user might want to show signatures with `font-lock-type-face` for instance, and have `lsp-lens-face` for various details. I do agree though that having a details face is a good idea, and that we should add one, making `lsp-signature-face` inherit from it.

Reply:
base details face sounds good.

Reply:
Great! It is now also used in `lsp-lens`. Lots of inheritance for styling is usually a good idea.